ios/Codeforces Watcher/Application/features/auth/signIn/SignInView.swift

```diff
@@ -0,0 +1,92 @@
+import SwiftUI
+
+struct SignInView: View {
+    var didSignInClick: ((String, String) -> Void)?
+    var onForgotPasswordTap: ((String) -> Void)?
+    var didSignUpClick: (() -> Void)?
+    
+    @State var email = ""
+    @State var emailView = ""
+    @State var password = ""
+    @State var passwordView = ""
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            Spacer()
+            
+            VStack(alignment: .leading, spacing: 44) {
+                Text("Sign In")
+                    .font(.system(size: 40, design: .monospaced))
+                
+                VStack(alignment: .leading, spacing: 24) {
+                    TextInputLayoutView(
+                        textReal: $email,
```

Comment:
Not sure that I get the difference between `textReal` and `textView`. Probably, not the best names.